include/wil/registry.h

```diff
@@ -2157,12 +2157,12 @@ namespace reg
             return {value};

```

Comment:
While you're here, care to make this a `return std::move(value)` ? IIRC, `std::optional<T>` has a ctor for `T&&`.

Reply:
Looks like this is a lesson on why brace construction is not always a good idea. `return value;` works as you'd expect & avoids the copy since `value` is an x-value. `return {value};` does _not_ avoid the copy since it's not an x-value in this context.

Example: https://godbolt.org/z/hxd4EfKWe